packages/features/ee/workflows/components/AgentConfigurationSheet.tsx

```diff
@@ -794,6 +885,204 @@ export function AgentConfigurationSheet({
                 )}
               </div>
             )}
+
+            {activeTab === "incomingCalls" && (
+              <div className="space-y-2">
+                {agentData?.outboundPhoneNumbers &&
+                agentData.outboundPhoneNumbers.filter(
+                  (phone) =>
+                    phone.subscriptionStatus === PhoneNumberSubscriptionStatus.ACTIVE ||
+                    !phone.subscriptionStatus
+                ).length > 0 ? (
+                  <>
+                    {!inboundAgentId ? (
+                      // Show setup button section when no inbound agent exists
+                      <div className="border-subtle rounded-xl border p-8">
+                        <div className="flex flex-col items-center space-y-6 text-center">
+                          <div className="bg-muted flex h-16 w-16 items-center justify-center rounded-lg">
+                            <Icon name="phone-incoming" className="text-subtle h-8 w-8" />
+                          </div>
+                          <div className="space-y-2">
+                            <h3 className="text-emphasis text-lg font-semibold">
```

Comment:
How about we create a different component for this? Seems like a pretty specific job of configuring incoming calls.